service/ui.go

```diff
@@ -276,7 +276,7 @@ func (uis *UIServer) GetServiceApp() *gimlet.APIApp {
 
 	// GraphQL
 	app.AddRoute("/graphql").Wrap(allowsCORS, needsLogin).Handler(playground.Handler("GraphQL playground", "/graphql/query")).Get()
-	app.AddRoute("/graphql/query").Wrap(allowsCORS, needsLogin).Handler(graphql.Handler()).Post().Get()
+	app.AddRoute("/graphql/query").Wrap(allowsCORS, needsLogin).Handler(graphql.Handler(evergreen.GetEnvironment().Settings().ApiUrl)).Post().Get()
```

Comment:
I believe that the ui server object has an environment and/or a settings object as a field that you can use rather than calling getEnvironment.